src/plasmapy/analysis/swept_langmuir/helpers.py

```diff
@@ -235,3 +237,285 @@ def sort_sweep_arrays(
         current = current[::-1]
 
     return voltage, current
+
+
+def _is_voltage_regularly_spaced(
```

Comment:
Does this function need to handle `numpy.nan` values?

Reply:
If the original `voltage` array has `np.nan` values, then `voltage_diff=np.diff(voltage)` will contain `np.nan` values to either side of the original `np.nan` location.  All the routines in the if-clause can handle `np.nan` values and would result in `is_regular_grid` being `False`, as it should be.

So, I do not think we need to handle it explicitly.